model/browser/events.yaml

```diff
@@ -0,0 +1,85 @@
+      It is recommended to be captured by using the PerformanceObserver API, or
+      when the window load event fires. If the page starts unloading before
+      the page loads, then the partial data can be captured by calling the
+      [performance.getEntriesByType][https://developer.mozilla.org/en-US/docs/Web/API/Performance/getEntriesByType]
+      method.
+
+      This event definition elevates a few key data points to body fields in
+      order to make it easier to access. In addition, the complete set of data
+      from the API may be sent in the body of the event as stringified JSON.
+    body:
+      id: browser.navigation_timing
+      requirement_level: required
+      stability: development
+      type: map
+      fields:
+        - id: name 
+          type: string
+          stability: development
+          brief: The resolved URL of the requested resource. 
+          requirement_level: recommended
+          examples: ["https://en.wikipedia.org/wiki/Main_Page"]
+        - id: fetchStart
+          type: double
+          stability: development
+          brief: >
+            The time immediately before the browser starts to fetch the
+            resource. The value is relative to the start of the navigation, and
+            it is measured in milliseconds.
+          requirement_level: recommended
+          examples: [9.600000000558794]
+        - id: unloadEventStart
+          type: double
+          stability: development
+          brief: >
+            The time immediately before the current document's unload event
+            handler starts. The value is relative to the start of the
+            navigation, and it is measured in milliseconds.
+          requirement_level: recommended
+          examples: [0.0]
+        - id: domInteractive
+          type: double
+          stability: development
+          brief: >
+            the time immediately before the user agent sets the document's
+            readyState to "interactive". The value is relative to the start of
+            the navigation, and it is measured in milliseconds.
+          requirement_level: recommended
+          examples: [147.70000000018626]
+        - id: domContentLoadedEventEnd
+          type: double
+          stability: development
+          brief: >
+             The time immediately after the current document's DOMContentLoaded
+             event handler completes. The value is relative to the start of the
+             navigation, and it is measured in milliseconds.
+          requirement_level: recommended
+          examples: [189.0]
+        - id: domComplete
+          type: double
+          stability: development
+          brief: >
+            The time immediately before the user agent sets the document's
+            readyState to "complete". The value is relative to the start of the
+            navigation, and it is measured in milliseconds.
```

Comment:
I'd suggest moving the second sentence to the `note` (to make the `brief` shorter)

and maybe add links to the properties in each case, to be super explicit that these are defined externally, e.g. https://www.w3.org/TR/navigation-timing-2/#dom-performancenavigationtiming-domcomplete